JustSaying/MessagingConfig.cs

```diff
@@ -24,25 +23,30 @@ public MessagingConfig()
         public int PublishFailureBackoffMilliseconds { get; set; }
         public Action<MessageResponse, Message> MessageResponseLogger { get; set; }
         public IReadOnlyCollection<string> AdditionalSubscriberAccounts { get; set; }
-        public IList<string> Regions { get; private set; }
+        public IList<string> Regions { get; }
         public Func<string> GetActiveRegion { get; set; }
         public IMessageSubjectProvider MessageSubjectProvider { get; set; }
 
         public virtual void Validate()
         {
             if (!Regions.Any() || string.IsNullOrWhiteSpace(Regions.First()))
             {
-                throw new ArgumentNullException("config.Regions", "Cannot have a blank entry for config.Regions");
+                throw new InvalidOperationException("Cannot have a blank entry for config.Regions");

```

Comment:
Maybe update the message to `$"Cannot have a blank entry for the {nameof(Regions)} property."`?